codersdk/provisionerdaemons.go

```diff
@@ -272,10 +272,12 @@ type ProvisionerKey struct {
 	OrganizationID uuid.UUID `json:"organization" table:"organization_id" format:"uuid"`
 	Name           string    `json:"name" table:"name,default_sort"`
 	// HashedSecret - never include the access token in the API response
+	Tags map[string]string `json:"tags" table:"tags"`
```

Comment:
Nit: move the `HashedSecret` comment to the end of the fields.

Reply:
I can do this in the next PR 👍

Reply:
had to rebase so doing it here now